plugins/janus_audiobridge.c

```diff
@@ -2846,10 +2874,79 @@ void janus_audiobridge_incoming_rtp(janus_plugin_session *handle, int video, cha
 			g_free(pkt);
 			return;
 		}
-		pkt->length = opus_decode(participant->decoder, payload, plen, (opus_int16 *)pkt->data, BUFFER_SAMPLES, USE_FEC);
-		g_atomic_int_set(&participant->decoding, 0);
+
+		/* Check sequence number received, verify if relevant to expected one. */
```

Comment:
Same here, this should all be done only if FEC is in use, I guess: unless there's a good reason to have it done for all cases?

Reply:
Same previous reply